doc/source/admin/cluster.md

```diff
@@ -146,31 +146,15 @@ Runs jobs via the [TORQUE Resource Manager](http://www.adaptivecomputing.com/pro
 
 #### Dependencies
 
-Galaxy uses the [pbs_python](https://oss.trac.surfsara.nl/pbs_python/) module to interface with TORQUE.  pbs_python must be compiled against your TORQUE installation, so it cannot be provided with Galaxy.  However, we provide all the necessary automation to compile it - ([more about Galaxy's Framework dependencies](framework_dependencies.html)):
+Galaxy uses the [pbs_python](https://github.com/ehiggs/pbs-python) module to interface with TORQUE.  pbs_python must be compiled against your TORQUE installation, so it cannot be provided with Galaxy. You can install the package as follows:
 
 ```console
-galaxy_user@galaxy_server% cd /clusterfs/galaxy/galaxy-app
-galaxy_user@galaxy_server% LIBTORQUE_DIR=/path/to/libtorque python scripts/scramble.py -e pbs_python
+galaxy_user@galaxy_server% git clone https://github.com/ehiggs/pbs-python
+galaxy_user@galaxy_server% cd pbs-python
+galaxy_user@galaxy_server% python setup.py install
+galaxy_user@galaxy_server% source /clusterfs/galaxy/galaxy-app/.venv/bin/activate
```

Comment:
Shouldn't this line be executed before the install?

Reply:
yep, thanks!